.codecov.yml

```diff
@@ -12,5 +12,8 @@ coverage:
   status:
     project:
       default:
-        target: 70%    # the required coverage value
-        threshold: 1%  # the leniency in hitting the target
+        target: auto
```

Comment:
Don't we want to keep it as 70?

Reply:
> Don't we want to keep it as 70?

Current coverage after this PR is merged would be 91.97%.  70 is way too low a threshold for the overall project.  Changing it to `auto` uses current project coverage as a baseline, with the threshold an acceptable level of lowering it.

I have added 70 in the section below as a minimum requirement for "patch", which means 70% of new lines must be covered.

See https://docs.codecov.com/docs/commit-status#patch-status